### PR TITLE
IAM Creative Specs & Safe Zone Updates

### PR DESCRIPTION
**Pull request by sarahwilson9**
# Pull Request/Issue Resolution

Cleaned up IAM Creative Details pages, and updated specs for new Image Safe Zone (releasing early access Tues 4/7). Details:
1. Moved Image Safe Zone to Full-Screen page, as it's only for that message type
2. Updated spec images and tables referencing image sizes on all 4 pages.
3. Lots of general content tweaking for clarity and accuracy.

### Is this change associated with a Braze feature/product release?
- [x] Yes (__4/7/20__)

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @KellieHawks as a reviewer when the your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details.md [hunk 1]
```diff
@@ -16,62 +17,75 @@ guide_featured_list:
...
 
-## Gifs & Videos
+{% endtab %}
+{% endtabs %}
+{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
+
+
+### Gifs & Videos
 
 Braze currently supports gifs for Web (included), [Android]({{ site.baseurl }}/developer_guide/platform_integration_guides/android/in-app_messaging/gifs/), and iOS (included) in-app messaging, given that it has been enabled during the desired platform integration. For more on video in in-app messages, see our [Customization documentation]({{ site.baseurl }}/user_guide/message_building_by_channel/in-app_messages/customize/#video).
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details.md:80]: Can you edit the android link to:

{{ site.baseurl }}/developer_guide/platform_integration_guides/android/in-app_messaging/customization/#gifs-IAMs

File: _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details/fullscreen.md [hunk 2]
```diff
@@ -8,37 +8,36 @@ page_order: 0
...
+- We only accept `PNG`, `JPG`, and `GIF` file types.
+- We recommend that your images be 500KB.
 
-<br>
-![Full-Screen Viewport][3b]{: style="max-width: 80%; border: none;"}
-<br>
+| Layout | Asset Size | Notes |
+|--- | --- | --- |
+| Image + Text | 6:5 aspect ratio<br>Hi-Res 1200 x 1000px<br> Min. 600 x 500px | Cropping can occur on all sides, but the image will always fill the top 50% of the viewport |
+| Image Only | 3:5 aspect ratio<br>Hi-Res 1200 x 2000px<br> Min. 600 x 1000px | Cropping can occur on the left and right edges on taller devices |
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details/fullscreen.md:29]: Can you add this line, {: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3} to the bottom of the chart? This piece makes sure the text doesn't wrap to the next line.

| Image Only | 3:5 aspect ratio<br>Hi-Res 1200 x 2000px<br> Min. 600 x 1000px | Cropping can occur on the left and right edges on taller devices |
{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}

File: _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details.md (hunk 1 above)
Comment (sarahwilson9) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details.md]: yep!

File: _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details/fullscreen.md (hunk 2 above)
Comment (sarahwilson9) [reply _docs/_user_guide/message_building_by_channel/in-app_messages/creative_details/fullscreen.md:29]: added

**Pull request merged**